--- a/Tools/ChangeLog	
+++ a/Tools/ChangeLog	
@@ -1, +1,13 @@ 
+2018-09-12  Michael Catanzaro  <mcatanzaro@igalia.com>
+
+        [WPE] fails to build on ARM arches complaining about undefined reference to `epoxy_eglMakeCurrent'
+        https://bugs.webkit.org/show_bug.cgi?id=189556
+
+        Reviewed by NOBODY (OOPS!).
+
+        This is only a speculative build fix as I didn't attempt to reproduce. Explicitly link to
+        libepoxy.
+
+        * wpe/backends/CMakeLists.txt:
+
 == Rolled over to ChangeLog-2018-09-11 ==
--- a/Tools/wpe/backends/CMakeLists.txt	
+++ a/Tools/wpe/backends/CMakeLists.txt	
@@ -21,6 +21,7 @@ set(WPEToolingBackends_INCLUDE_DIRECTORIES
 set(WPEToolingBackends_SYSTEM_INCLUDE_DIRECTORIES
     ${CAIRO_INCLUDE_DIRS}
     ${GLIB_INCLUDE_DIRS}
+    ${LIBEPOXY_INCLUDE_DIRS}
     ${WPE_INCLUDE_DIRS}
     ${WPEBACKEND_FDO_INCLUDE_DIRS}
 )
@@ -28,6 +29,7 @@ set(WPEToolingBackends_SYSTEM_INCLUDE_DIRECTORIES
 set(WPEToolingBackends_LIBRARIES
     ${CAIRO_LIBRARIES}
     ${GLIB_LIBRARIES}
+    ${LIBEPOXY_LIBRARIES}
     ${LIBXKBCOMMON_LIBRARIES}
     ${WAYLAND_LIBRARIES}
     ${WPE_LIBRARIES}
