diff --git a/Tools/wpe/backends/CMakeLists.txt b/Tools/wpe/backends/CMakeLists.txt
index f9e8b43b9fb9a44de769fd95c11d70276519c505..439e8e3f48f489dfb35116e6a5c90b654fa51bf9 100644
--- a/Tools/wpe/backends/CMakeLists.txt
+++ b/Tools/wpe/backends/CMakeLists.txt
@@ -21,6 +21,7 @@ set(WPEToolingBackends_INCLUDE_DIRECTORIES
 set(WPEToolingBackends_SYSTEM_INCLUDE_DIRECTORIES
     ${CAIRO_INCLUDE_DIRS}
     ${GLIB_INCLUDE_DIRS}
+    ${LIBEPOXY_INCLUDE_DIRS}
     ${WPE_INCLUDE_DIRS}
     ${WPEBACKEND_FDO_INCLUDE_DIRS}
 )
@@ -28,6 +29,7 @@ set(WPEToolingBackends_SYSTEM_INCLUDE_DIRECTORIES
 set(WPEToolingBackends_LIBRARIES
     ${CAIRO_LIBRARIES}
     ${GLIB_LIBRARIES}
+    ${LIBEPOXY_LIBRARIES}
     ${LIBXKBCOMMON_LIBRARIES}
     ${WAYLAND_LIBRARIES}
     ${WPE_LIBRARIES}
